<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-08-30T19:23:48 IST" name="Default suite" finished-at="2024-08-30T19:24:01 IST" duration-ms="13261">
    <groups>
    </groups>
    <test started-at="2024-08-30T19:23:48 IST" name="Default test" finished-at="2024-08-30T19:24:01 IST" duration-ms="13261">
      <class name="Pkg.Class1">
        <test-method is-config="true" signature="BrowserInvoke()[pri:0, instance:Pkg.Class1@38c6f217]" started-at="2024-08-30T19:23:48 IST" name="BrowserInvoke" finished-at="2024-08-30T19:24:01 IST" duration-ms="12622" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BrowserInvoke -->
        <test-method signature="emailClick()[pri:0, instance:Pkg.Class1@38c6f217]" started-at="2024-08-30T19:24:01 IST" name="emailClick" finished-at="2024-08-30T19:24:01 IST" duration-ms="581" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected:<5> but was:<>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected:<5> but was:<>
at org.junit.Assert.fail(Assert.java:89)
at org.junit.Assert.failNotEquals(Assert.java:835)
at org.junit.Assert.assertEquals(Assert.java:120)
at org.junit.Assert.assertEquals(Assert.java:146)
at Pkg.Class1.emailClick(Class1.java:98)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- emailClick -->
      </class> <!-- Pkg.Class1 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
